zeroex/ordervalidator/order_validator.go

```diff
@@ -335,6 +340,12 @@ func (o *OrderValidator) BatchValidate(rawSignedOrders []*zeroex.SignedOrder, ar
 				ctx, cancel := context.WithTimeout(context.Background(), getOrderRelevantStateTimeout)
 				defer cancel()
 				opts := &bind.CallOpts{
+					// HACK(albrow): From field should not be required for eth_call but
+					// including it here is a workaround for a bug in Ganache. Removing
+					// this line causes Ganache to crash.
+					// TODO(albrow): Test if this causes problems for real Ethereum nodes
```

Comment:
@fabioberger I think you mentioned this shouldn't cause any problems for real Ethereum nodes. If that's the case we should remove this `TODO` (but keep the `HACK` comment).